Easydict/Feature/ViewController/Storage/EZLocalStorage.m

```diff
@@ -141,6 +141,20 @@ - (void)setAllServiceTypes:(NSArray<EZServiceType> *)allServiceTypes windowType:
     return allServices;
 }
 
+- (EZQueryService *)service:(EZServiceType)serviceType windowType:(EZWindowType)windowType {
+    EZQueryService *service = [EZServiceTypes.shared serviceWithType:serviceType];
+    EZServiceInfo *serviceInfo = [self serviceInfoWithType:service.serviceType windowType:windowType];
+    BOOL enabled = YES;
+    BOOL enabledQuery = YES;
+    if (serviceInfo) {
+        enabled = serviceInfo.enabled;
+        enabledQuery = serviceInfo.enabledQuery;
+    }
+    service.enabled = enabled;
+    service.enabledQuery = enabledQuery;
+    return service;
+}
+
```

Comment:
If you want to add this kind of repetitive code, it is recommended to refactor it as well.

```objc
- (NSArray<EZQueryService *> *)allServices:(EZWindowType)windowType {
    NSArray *allServices = [EZServiceTypes.shared servicesFromTypes:[self allServiceTypes:windowType]];
    for (EZQueryService *service in allServices) {
        EZServiceInfo *serviceInfo = [self serviceInfoWithType:service.serviceType windowType:windowType];
        BOOL enabled = YES;
        BOOL enabledQuery = YES;
        if (serviceInfo) {
            enabled = serviceInfo.enabled;
            enabledQuery = serviceInfo.enabledQuery;
        }
        service.enabled = enabled;
        service.enabledQuery = enabledQuery;
    }
    return allServices;
}

- (EZQueryService *)service:(EZServiceType)serviceType windowType:(EZWindowType)windowType {
    EZQueryService *service = [EZServiceTypes.shared serviceWithType:serviceType];
    EZServiceInfo *serviceInfo = [self serviceInfoWithType:service.serviceType windowType:windowType];
    BOOL enabled = YES;
    BOOL enabledQuery = YES;
    if (serviceInfo) {
        enabled = serviceInfo.enabled;
        enabledQuery = serviceInfo.enabledQuery;
    }
    service.enabled = enabled;
    service.enabledQuery = enabledQuery;
    return service;
}
```